### ISSUE TITLE
The default value of evictionRules is project-scoped

### ISSUE THREAD

**Issue by julienrf** [scalacenter/sbt-eviction-rules#4]
I tried to add:

~~~ scala
inThisBuild(Seq(
  evictionRules ++= Seq(
    "org.scala-lang.modules" %% "*" % "semver"
  )
))
~~~

Because I would like to define eviction rules once in my build and benefit from it in every module.

This does not work currently, I get a “reference to an undefined setting” error.

I think this issue could be solved by scoping the default value of `evictionRules` to `ThisBuild` (by using `buildSettings` instead of `projectSettings`).

As a workaround, I can use `:=` instead of `++=`.

**Comment by julienrf** [comment]
Actually, my workaround doesn’t work as such eviction rules wouldn’t be picked by `evictionWarnings`.

**Comment by julienrf** [comment]
Fixed by #5
